spec/fabricators/video_fabricator.rb

```diff
@@ -1,4 +1,8 @@
 Fabricator(:video) do
   title { Faker::Lorem.characters(5) }
   description { Faker::Lorem.sentence }
-end
\ No newline at end of file
+end
+
+Fabricator(:video_upload, from: :video) do
+  large_cover { Rack::Test::UploadedFile.new("./spec/support/uploads/monk_large.jpg") }
```

Comment:
Thanks for that info, I changed my code:
```
Fabricator(:video_upload, from: :video) do
  large_cover { File.open("./spec/support/uploads/monk_large.jpg") }
end
```
I'll keep an eye on that for later tests